src/processTargets/processSelectionType.ts

```diff
@@ -44,6 +44,13 @@ export default function (
         selection,
         selectionContext
       );
+    case "nonwhiteSpaceOrDelimiterSequence":
```

Comment:
Since all three of these call the same function with a different regex, I wonder if the case should call through on 2 of the 3 and then then in the third, we look up the regex from a hash. Alternative would be to have the modifier as a class that has a default regex stored on it, and use that.

Reply:
This is setup is going to change a lot when big PR merges in, so during conflict resolution it will in fact end up close to what you're proposing